src/Configurator/BundlesConfigurator.php

```diff
@@ -90,6 +90,14 @@ private function dump(string $file, iterable $bundles): void
         }
 
         file_put_contents($file, $contents);
+
+        if (function_exists('opcache_invalidate')) {
+            opcache_invalidate($file);
+        }
+
+        if (function_exists('apc_delete_file')) {
+            apc_delete_file($file);
```

Comment:
APCu replaces the user caching part of APC, not the opcode cache part (as opcode caching is handled by OPCache).

And yes I'm sure it does not work with PHP 7. Extensions require a rewrite for PHP 7, especially when they are as low level as an opcode cache. APC would not even compile on PHP 7.